### ISSUE TITLE
Divi's lightbox feature supersedes SLB functionality.

### ISSUE THREAD

**Issue by Xyn333** [archetyped/simple-lightbox#496]
So in trying to get the SLB running on a slick site to make the site better with divi theme, the bastard divi lightbox doesnt fall away and overlays the SLB , so two pop up, though the SLB works and the DIVI lightbox, ofcourse having just one is the way its supposed to be.

It would be epic if this gets a fix as SLB has been the goto for lightboxing gallery/images disconnected across page.

The example is localhost so I cnt link it. sorry.. but please try it with divi to see,

**Comment by archetyped** [comment]
Hi, it looks like the Divi theme's lightbox can be disabled via the theme's settings (reference: [Divi Documentation: Image Module](http://www.elegantthemes.com/gallery/divi/documentation/image/)).  Have you tried this?  Does your issue still persist?

**Comment by Xyn333** [comment]
Yes which then makes all click functionality go away , it doesn't work at
all then

On Wednesday, 06 July 2016, Archetyped <notifications@github.com> wrote:

> Hi, it looks like the Divi theme's lightbox can be disabled via the
> theme's settings (reference: Divi Documentation: Image Module
> <http://www.elegantthemes.com/gallery/divi/documentation/image/>). Have
> you tried this? Does your issue still persist?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/archetyped/simple-lightbox/issues/496#issuecomment-230661043>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGqoHdX86ISEfv0qA7L1icM0kQYlI1-pks5qSx1ugaJpZM4JCzzA>
> .
>


-- 
Kind regards​



_______________________________

​ASPIRE TO MORE
_______________________________

[+27] 079 612 24 43
_______________________________

info@xyndigital.com
_______________________________

<http://facebook.com/xyn333> <https://www.instagram.com/xyndigital/>
<https://twitter.com/christynpienaar> <https://plus.google.com/+XynPihnar>

**Comment by Xyn333** [comment]
Hey,

I just signed the petition "European Parliament: Stop the
censorship-machinery! Save the Internet!" and wanted to see if you could
help by adding your name.

Our goal is to reach 4,180,952 signatures and we need more support. You can
read more and sign the petition here:

http://chng.it/KmCL5T87X8

Thanks!
Christyn
